katalog/grafana/deploy.yml

```diff
@@ -47,6 +49,54 @@ spec:
         - mountPath: /grafana-dashboard-definitions/0/k8s
           name: grafana-dashboard-definitions-k8s
           readOnly: false
```

Comment:
Isn't this redundant?

Reply:
No, `grafana-dashboard-definitions-k8s` includes: 
- coredns.json
- etcd.json

dashboards. 

Generated in kustomization.yaml file